### ISSUE TITLE
leak astropy broken on Python 3.6

### ISSUE THREAD

**Issue by cdeil** [bmwant/leak#3]
I'm getting this error on Python 3.6 for `leak astropy`:
```
$ leak astropy
================================================================================
astropy
Community-developed python astronomy tools
--------------------------------------------------------------------------------
Author:              The Astropy Developers
Author mail:         astropy.team@gmail.com
Available versions:  52
Home page:           http://astropy.org
================================================================================
Traceback (most recent call last):
  File "/Users/deil/Library/Python/3.6/lib/python/site-packages/leak/main.py", line 104, in main
    versions = sorted(releases.keys(), reverse=True, key=StrictVersion)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/version.py", line 40, in __init__
    self.parse(vstring)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/version.py", line 137, in parse
    raise ValueError("invalid version number '%s'" % vstring)
ValueError: invalid version number '1.2rc1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/deil/Library/Python/3.6/bin/leak", line 4, in <module>
    cli.cli()
  File "/Users/deil/Library/Python/3.6/lib/python/site-packages/leak/cli.py", line 12, in cli
    main.main(package_name=args.package_name)
  File "/Users/deil/Library/Python/3.6/lib/python/site-packages/leak/main.py", line 108, in main
    versions = sorted(releases.keys(), reverse=True, key=splitter)
TypeError: '<' not supported between instances of 'map' and 'map'
```
Can you please make it work?

**Comment by bmwant** [comment]
@cdeil I will add travis CI to this package and will test leak on different python versions, so compatibility issues should be resolved

**Comment by bmwant** [comment]
@cdeil fixed in `1.1.0`

**Comment by cdeil** [comment]
Thanks!

Just to confirm, this works for me now on Python 3.6.
